runelite-client/src/main/java/net/runelite/client/plugins/timestamp/TimestampPlugin.java

```diff
@@ -76,10 +87,14 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 		final ZonedDateTime time = ZonedDateTime.ofInstant(
 			Instant.ofEpochSecond(messageNode.getTimestamp()), ZoneId.systemDefault());
 
-		final String dateFormat = time.get(ChronoField.HOUR_OF_DAY) + ":" +
-			String.format("%02d", time.get(ChronoField.MINUTE_OF_HOUR));
-
-		String timestamp = "[" + dateFormat + "] ";
+		String timestamp = config.timestampFormat()
```

Comment:
```suggestion
		final String timestamp = config.timestampFormat()
```

Reply:
you probably don't really care anymore but this was wrong